### ISSUE TITLE
Weird display of the highlighted blocks

### ISSUE THREAD

**Issue by beurdouche** [FStarLang/fstar-mode.el#124]
I have updated my version of emacs to 27.1 recently and now get the following behavior...

<img width="823" alt="Screenshot 2021-03-20 at 19 24 20" src="https://user-images.githubusercontent.com/1193431/111881771-695fc480-89b2-11eb-9c84-d6cca861a0b2.png">

@cpitclaudel Is there a way for the highlighting to cover the length of the frame? :) Thanks

**Comment by cpitclaudel** [comment]
Yep, that's a new emacs "feature": faces that want to extend to the end of the line need a `:extend` spec.  Can you try adding `:extend t` to the relevant `defface` definitions? (it shouldn't break older Emacsen too much)
